src/nlp/builder.py

```diff
@@ -743,39 +775,22 @@ def _build_pcollection(pipeline, extracted_dir):
         """
         raise NotImplementedError()
 
-    def _as_dataset(self, split: Split = Split.TRAIN):
```

Comment:
We don't use the parquet reader anymore as we can't do memory mapping on parquet